.gitignore

```diff
@@ -0,0 +1 @@
+*.pyc
```

Comment:
@Ali-Ad add the following in the file 
.idea/*
*.gitignore
*__pycache__*
__pycache__/*
*/__pycache__/*
*/*/__pycache__/*
*.pyc